server/cpho/jinja2/login.jinja2

```diff
@@ -2,63 +2,79 @@
 {% block title %}{{ tm("log_in") }}{% endblock %}
 {% block content_breadcrumbs_container %}
   {# override to remove <nav> tag #}
-{% endblock content_breadcrumbs_container %}
+{% endblock %}
 {% block content %}
   <div class="container p-0 d-flex justify-content-center"
        id="login-container">
     <div class="col-md-6 col-sm-12">
-      <form method="post"
-            action="{{ request.get_full_path() }}"
-            autocomplete="off">
-        {{ csrf_input }}
-        {% if next %}
-          <div class="alert alert-warning fade show">
-            <span>{{ tm("please_login_to_see_page") }}</span>
-          </div>
-        {% endif %}
-
-        {% if form.errors %}
-          <div class="alert alert-danger alert-dismissible fade show">
-            <span>{{ tm("bad_user_name_and_password") }}</span>
-            <button type="button"
-                    class="btn-close"
-                    data-bs-dismiss="alert"
-                    aria-label="{{ tm("close") }}"></button>
-          </div>
-        {% endif %}
-
+      {% if next %}
+        <div class="alert alert-warning fade show">
+          <span>{{ tm("please_login_to_see_page") }}</span>
+        </div>
+      {% endif %}
+      {% if PHAC_ASPC_OAUTH_PROVIDER == "microsoft" %}
         <div class="card">
-          <div class="h2" class="card-header">{{ tm("log_in") }}</div>
+          <h2 class="card-header">{{ tm("log_in") }}</h2>
           <div class="card-body">
-            <div class="form-floating mb-3">
-              <input type="text"
-                     name="{{ form.username.name }}"
-                     maxlength="150"
-                     placeholder="{{ tm("username") }}"
-                     required
-                     class="textinput textInput form-control">
-              <label for="{{ form.username.id_for_label }}">{{ form.username.label_tag() }}</label>
-            </div>
-            <div class="form-floating mb-3">
-              <input type="password"
-                     name="{{ form.password.name }}"
-                     maxlength="128"
-                     placeholder="{{ tm("password") }}"
-                     required
-                     class="textinput textInput form-control">
-              <label for="{{ form.password.id_for_label }}">{{ form.password.label_tag() }}</label>
+            <p>{{ tdt("Welcome") }}</p>
+            <p>{{ tm("prod_login_intro_2") }}</p>
+            {#
+              <p>
+                {{ tm("prod_login_intro_3") }}
+                <a href="{{ url("user_agreement") }}">{{ tm("user_agreement") | lower }}</a>.
+              </p>
+            #}
```

Comment:
Could delete this, and the corresponding translation entry, if you don't expect to need a user agreement on this project. Don't know if ITAP has it because the client requested it, or if it was required during approvals (and therefore is needed here too).

Reply:
Go ahead and delete it, let's try and slip by without it, ITAP was special